src/Spectre.Service.Tests/DataRootConfigTest.cs

```diff
@@ -0,0 +1,45 @@
+﻿using NUnit.Framework;
+using Spectre.Service.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Spectre.Service.Tests
+{
+    [TestFixture]
+    class DataRootConfigTest
+    {
+        private string goodLocalPath  = @"\Path\To\Data";
+        private string wrongLocalPath = @"\Path\To\*";
```

Comment:
We can. This is a [complicated issue](https://stackoverflow.com/a/1976050). Should we list all the possibilities here instead of testing one of the most obvious cases and trusting `FileInfo` with the rest?